litmus-portal/frontend/src/views/Settings/TeammingTab/tableData.tsx

```diff
@@ -0,0 +1,218 @@
+import { useMutation } from '@apollo/client/react/hooks';
+import { Avatar, IconButton, TableCell, Typography } from '@material-ui/core';
+import moment from 'moment';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import ButtonFilled from '../../../components/Button/ButtonFilled';
+import ButtonOutline from '../../../components/Button/ButtonOutline';
+import Loader from '../../../components/Loader';
+import Unimodal from '../../../containers/layouts/Unimodal';
+import { REMOVE_INVITATION } from '../../../graphql/mutations';
+import { GET_USER } from '../../../graphql/quries';
+import { MemberInvitation } from '../../../models/graphql/invite';
+import { Member } from '../../../models/graphql/user';
+import { RootState } from '../../../redux/reducers';
+import userAvatar from '../../../utils/user';
+import useStyles from './styles';
+
+interface TableDataProps {
+  row: Member;
+  index: number;
+}
+const TableData: React.FC<TableDataProps> = ({ row /* index */ }) => {
+  const classes = useStyles();
+  // const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  // const [role, setRole] = useState<string>('Viewer');
+
+  const userData = useSelector((state: RootState) => state.userData);
+
+  // for closing the menu option
+  // const handleClose = () => {
+  //   setAnchorEl(null);
+  // };
+  const [open, setOpen] = React.useState(false);
+  // Function to display date in format Do MMM,YYYY Hr:MM AM/PM
+  const formatDate = (date: string) => {
+    const day = moment(date).format('Do MMM, YYYY LT');
+    return day;
+  };
+
+  // mutation to remove member
+  const [removeMember, { loading }] = useMutation<MemberInvitation>(
+    REMOVE_INVITATION,
+    {
+      onCompleted: () => {
+        setOpen(false);
+      },
+      onError: () => {},
+      refetchQueries: [
+        { query: GET_USER, variables: { username: userData.username } },
+      ],
+    }
+  );
+
+  return (
+    <>
+      <TableCell className={classes.firstTC} component="th" scope="row">
+        <div className={classes.firstCol}>
+          <Avatar
+            data-cy="avatar"
+            alt="User"
+            className={classes.avatarBackground}
+          >
+            {row?.name ? userAvatar(row?.name) : userAvatar(row?.user_name)}
+          </Avatar>
+          {row?.name}
+        </div>
+      </TableCell>
+      <TableCell className={classes.otherTC}>
+        {row?.role}
+        {/* {row?.role === 'editor' || row?.role === 'viewer' ? (
```

Comment:
Do we need these commented lines of code in future? Else this can be removed.

Reply:
Yes, it will be used in the next release

Reply:
same